### ISSUE TITLE
fail to start new project 

### ISSUE THREAD

**Issue by ri7nz** [jihchi/app-template-rescript-react#11]
Hello @jihchi , I have been try to use this template.

This is the following what I'm doing (following on readme)
```shell
❯ npx create-snowpack-app new-dir --template app-template-reason-react --use-yarn
Need to install the following packages:
  create-snowpack-app
Ok to proceed? (y) y

  - Using template app-template-reason-react
  - Creating a new project in /Users/r17/evl/new-dir
  - Installing package dependencies. This might take a couple of minutes.

warning package.json: No license field
warning No license field
warning snowpack > rollup > fsevents@2.1.3: "Please update to latest v2.3 or v2.2"
warning snowpack > pacote > @npmcli/run-script > node-gyp > request@2.88.2: request has been deprecated, see https://github.com/request/request/issues/3142
warning snowpack > pacote > @npmcli/run-script > node-gyp > request > har-validator@5.1.5: this library is no longer supported
warning " > @rescript/react@0.10.2" has unmet peer dependency "bs-platform@>=8.3.0".

  - Initializing git repo.

  - Success!

Quickstart:

  cd new-dir
  yarn start

All Commands:

  yarn install     Install your dependencies. (We already ran this one for you!)
  yarn start       Start your development server.
  yarn run build   Build your website for production.
  yarn test        Run your tests.

❯ cd new-dir
❯ yarn run start
yarn run v1.22.5
warning package.json: No license field
$ snowpack dev
[24:60:40] [snowpack] Welcome to Snowpack! Because this is your first time running
this project, Snowpack needs to prepare your dependencies. This is a one-time step
and the results will be cached for the lifetime of your project. Please wait...
[24:60:40] [snowpack] Ready!
[24:60:40] [snowpack] Server started in 12ms.
[24:60:40] [snowpack] Local: http://localhost:8080
[24:60:40] [snowpack] Network: http://192.168.0.11:8080
[24:60:40] [npm] npm
[24:60:40] [npm] ERR! No workspaces found:
[24:60:40] [npm] npm ERR!   --workspace=true
[24:60:40] [npm] npm ERR! A complete log of this run can be found in:
[24:60:40] [npm] npm ERR!     $HOME/.npm/_logs/2021-04-17T16_59_40_507Z-debug.log
[24:60:40] [npm] Command exited with error code: Error: Command failed with exit code 1: npm run build:res -w
npm ERR! No workspaces found:
npm ERR!   --workspace=true

npm ERR! A complete log of this run can be found in:
npm ERR!     $HOME/.npm/_logs/2021-04-17T16_59_40_507Z-debug.log
error Command failed with exit code 1.
info Visit https://yarnpkg.com/en/docs/cli/run for documentation about this command.
```
So, I'm check `debug.log` 
```shell
0 verbose cli [
0 verbose cli   '/opt/homebrew/Cellar/node/15.14.0/bin/node',
0 verbose cli   '/opt/homebrew/bin/npm',
0 verbose cli   'run',
0 verbose cli   'build:res',
0 verbose cli   '-w'
0 verbose cli ]
1 info using npm@7.7.6
2 info using node@v15.14.0
3 timing npm:load:whichnode Completed in 0ms
4 timing config:load:defaults Completed in 1ms
5 timing config:load:file:/opt/homebrew/lib/node_modules/npm/npmrc Completed in 1ms
6 timing config:load:builtin Completed in 1ms
7 timing config:load:cli Completed in 1ms
8 timing config:load:env Completed in 1ms
9 timing config:load:file:$HOME/evl/new-dir/.npmrc Completed in 0ms
10 timing config:load:project Completed in 1ms
11 timing config:load:file:$HOME/.npmrc Completed in 0ms
12 timing config:load:user Completed in 0ms
13 timing config:load:file:/opt/homebrew/etc/npmrc Completed in 0ms
14 timing config:load:global Completed in 0ms
15 timing config:load:validate Completed in 1ms
16 timing config:load:setEnvs Completed in 0ms
17 timing config:load Completed in 6ms
18 timing npm:load:configload Completed in 6ms
19 timing npm:load:setTitle Completed in 2ms
20 timing npm:load:setupLog Completed in 1ms
21 timing npm:load:cleanupLog Completed in 1ms
22 timing npm:load:configScope Completed in 0ms
23 timing npm:load:projectScope Completed in 0ms
24 timing npm:load Completed in 10ms
25 timing config:load:flatten Completed in 1ms
26 timing command:run-script Completed in 1ms
27 verbose stack Error: No workspaces found:
27 verbose stack   --workspace=true
27 verbose stack     at getWorkspaces (/opt/homebrew/lib/node_modules/npm/lib/workspaces/get-workspaces.js:27:11)
27 verbose stack     at async RunScript.runWorkspaces (/opt/homebrew/lib/node_modules/npm/lib/run-script.js:191:24)
28 verbose cwd $HOME/evl/new-dir
29 verbose Darwin 20.3.0
30 verbose argv "/opt/homebrew/Cellar/node/15.14.0/bin/node" "/opt/homebrew/bin/npm" "run" "build:res" "-w"
31 verbose node v15.14.0
32 verbose npm  v7.7.6
33 error No workspaces found:
33 error   --workspace=true
34 verbose exit 1
```

I don't know why shown `--workspace=true`, but I assume this is about command args `-w`.

thank you

**Comment by jihchi** [comment]
I've merged the PR #12 and published version [`v3.0.3`](https://github.com/jihchi/app-template-rescript-react/releases/tag/v3.0.3).

Please have a try and let me know if it does not work.

**Comment by ri7nz** [comment]
@jihchi  Thank you jihchi, I'm close this is issue.
